src/main/java/org/ngafid/ProcessUpload.java

```diff
@@ -51,9 +50,9 @@ public class ProcessUpload {
     private static final Map<String, FileProcessor> fileProcessors = new HashMap<>();
 
     static {
-        fileProcessors.put(".csv", null);
-        fileProcessors.put(".json", null);
-        fileProcessors.put(".gpx", null);
+        fileProcessors.put(".csv", new CSVFileProcessor());
```

Comment:
We can probably just change this to:

`Map.of(".csv", new CSVFileProcessor(), ...)`

so the declaration and definition can be together.